Marshmallow 3.0.0rc3 incompatibility

Looks like marshmallow changed out from under marshmallow-peewee:

```
ImportError while loading conftest 'tests/conftest.py'.
[...]
.venv/lib/python3.7/site-packages/marshmallow_peewee/__init__.py:9: in <module>
    from .schema import ModelSchema # noqa
.venv/lib/python3.7/site-packages/marshmallow_peewee/schema.py:5: in <module>
    from .convert import ModelConverter
.venv/lib/python3.7/site-packages/marshmallow_peewee/convert.py:7: in <module>
    from .fields import ForeignKey
.venv/lib/python3.7/site-packages/marshmallow_peewee/fields.py:5: in <module>
    from marshmallow.compat import PY2, string_types
E   ImportError: cannot import name 'string_types' from 'marshmallow.compat' (.venv/lib/python3.7/site-packages/marshmallow/compat.py)
```

https://github.com/marshmallow-code/marshmallow/commit/d9bcc48317534aa9d7910ab5fc82d1541d5d2b24 is the change.

--- a/marshmallow_peewee/fields.py
+++ b/marshmallow_peewee/fields.py
@@ -1,10 +1,15 @@
 import datetime as dt
 
 from marshmallow import fields
-
-from marshmallow.compat import PY2, string_types
+from marshmallow.compat import PY2
 
 from . import MA_VERSION
+
+
+if PY2:
+    string_types = (str, unicode)
+else:
+    string_types = (str,)
 
 
 class Timestamp(fields.Field):

